pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -80,6 +80,17 @@ read_10x_files <- function(config, input_dir) {
       counts <- counts[[slot]]
     }
 
+    # Check if there are any rows with empty gene symbol in the count
+    # matrix and remove them if < 0.1% of the total number of features
+    # if more than 1 empty gene is present in the feature file,
+    # the count matrix will have the first rowname empty, while the 
+    # following will be ".1", ".2". etc...
```

Comment:
```suggestion
    # Check existence of empty gene symbols in count matrix' rownames.
    # If there are more than one the first will be empty, while the 
    # following will be ".1", ".2"... because Seurat runs make.unique
```